### DISCUSSION TITLE
How about including the preview contents in the email notification?

### DISCUSSION THREAD

**Discussion by unjeong-otsk** [outline/outline#1997] [Ideas]
Dear Outline Product Manager,

I am so big fan of your product "Outline" and fully use it for my team members as a Wiki page.
During using this, I came up with one idea which we can show the preview of contents in email notification before opening the Wiki page.

The below is a sample of the email notification from Google Workplace.

**1) A current Outline email notification**

![Outline](https://user-images.githubusercontent.com/63595064/113081382-8ba7dc80-9213-11eb-9dc3-9d472a80a2ca.PNG)

**2) Workplace email notification**

![Workplace](https://user-images.githubusercontent.com/63595064/113081390-8fd3fa00-9213-11eb-8126-856eb4025437.PNG)

**Comment by tommoor** [comment]
Hey, thanks for the post – we agree and are tracking this issue here:

https://github.com/outline/outline/issues/1803
